spec/generator/build_phase/compile_xcdatamodel_build_phase_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+module Xcake
+  module Generator
+    describe CompileXCDataModelBuildPhase do
+
+      before :each do
+        @node = double("Node").as_null_object
+      end
+
+      it "should not able to install folder" do
+        folder_path = 'Folder/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.